React/Views/ScrollView/RCTScrollContentShadowView.m

```diff
@@ -9,10 +9,30 @@
 
 #import <yoga/Yoga.h>
 
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+#import "RCTScrollContentLocalData.h"
+#endif // ]TODO(macOS ISS#2323203)
+
 #import "RCTUtils.h"
 
 @implementation RCTScrollContentShadowView
 
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+- (void)setLocalData:(RCTScrollContentLocalData *)localData
+{
+  RCTAssert(
+      [localData isKindOfClass:[RCTScrollContentLocalData class]],
+      @"Local data object for `RCTScrollContentView` must be `RCTScrollContentLocalData` instance.");
+
+  UIEdgeInsets insets = localData.insets;
+
+  super.marginRight = (YGValue){insets.right, YGUnitPoint};
```

Comment:
This fix is brilliant!  I tested it locally and everything looks perfect.   The only nit I have is to be left-to-right and right-to-left agnostic and use `marginEnd` here instead of `marginRight`.   I tested it in RTL mode, and it still works with `marginRight`, but I still think it would be more semantically correct to use `marginEnd`.

Also, it's not necessary to use a `UIEdgeInsets` on the local data since we only need two `CGFloat` values for `marginEnd` and `marginBottom`.   Recommend change the `RCTScrollContentLocalData` to just have two `CGFloat` type props.

<img width="752" alt="rtl" src="https://user-images.githubusercontent.com/30053638/85182099-8c4f2380-b23c-11ea-9a9d-d128f41fcefe.png">

Reply:
There's still a bug here that I haven't figured out yet. When RNTester starts, trying to grab the vertical scroll indicator also triggers a click on the element under it.